api/checks/webhook.go

```diff
@@ -212,18 +212,19 @@ func handlePullRequestEvent(ctx context.Context, payload []byte) (bool, error) {
 	return false, nil
 }
 
-func completeChecksForExistingRuns(ctx context.Context, sha string, products ...shared.ProductSpec) (bool, error) {
+func scheduleProcessingForExistingRuns(ctx context.Context, sha string, products ...shared.ProductSpec) (bool, error) {
 	// Jump straight to completed check_run for already-present runs for the SHA.
 	products = shared.ProductSpecs(products).OrDefault()
 	runsByProduct, err := shared.LoadTestRuns(ctx, products, nil, sha[:10], nil, nil, nil, nil)
 	if err != nil {
 		return false, fmt.Errorf("Failed to load test runs: %s", err.Error())
 	}
 	createdSome := false
+	api := NewSuitesAPI(ctx)
 	for _, rbp := range runsByProduct {
 		if len(rbp.TestRuns) > 0 {
-			created, err := completeCheckRun(ctx, sha, rbp.Product)
-			createdSome = createdSome || created
+			err := api.ScheduleResultsProcessing(sha, rbp.Product)
+			createdSome = createdSome || err == nil
```

Comment:
Hmm, I don't think you need `createdSome` any more. In fact, this method probably only need to return `error` now that you defer the processing to TaskQueue.

Reply:
It affects the content of the HTTP response (as to whether anything was scheduled or not).